<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIUserInterface</class>
 <widget class="QMainWindow" name="GUIUserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>580</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Don't GitHub, Use Procrastinate</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="modeSelector">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="maintenanceTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Maintenance</string>
       </attribute>
       <widget class="QListWidget" name="pendFileNames">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>60</y>
          <width>441</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addFilesButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>221</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Add WikiBooks Files</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>59</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pending:</string>
        </property>
       </widget>
       <widget class="QListWidget" name="currFileNames">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>180</y>
          <width>441</width>
          <height>81</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>160</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Currently in Index:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>280</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Data Structure:</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="hashRadio">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>310</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Hash Table</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="avlRadio">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>310</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>AVL Tree</string>
        </property>
       </widget>
       <widget class="QPushButton" name="parseDataButton">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>420</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Parse Data</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clearAllButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>420</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear All</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>420</y>
          <width>121</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Persistent</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="interactiveTab">
       <attribute name="title">
        <string>Interactive</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Boolean Query</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="queryText">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>541</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="searchQuery">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>26</y>
          <width>85</width>
          <height>32</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="tesultsTableWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>760</width>
          <height>391</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>160</number>
        </attribute>
        <column>
         <property name="text">
          <string>I.D.</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Author</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>TF/IDF</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="getStatsbutton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>480</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Statistics</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
